metricbeat/module/kibana/stats/stats.go

```diff
@@ -162,26 +157,12 @@ func (m *MetricSet) fetchStats(r mb.ReporterV2, now time.Time) error {
 		origURI := m.statsHTTP.GetURI()
 		defer m.statsHTTP.SetURI(origURI)
 
-		shouldCollectUsage := m.shouldCollectUsage(now)
-		m.statsHTTP.SetURI(origURI + "&exclude_usage=" + strconv.FormatBool(!shouldCollectUsage))
-
-		content, err = m.statsHTTP.FetchContent()
-		if err != nil {
-			if shouldCollectUsage {
-				// When errored in collecting the usage stats it may be counterproductive to try again on the next poll, try to collect the stats again after usageCollectionBackoff
-				m.usageNextCollectOn = now.Add(usageCollectionBackoff)
-			}
-			return err
-		}
+		m.statsHTTP.SetURI(origURI + "&exclude_usage=true")
+	}
```

Comment:
With this change we no longer need the comment above `if m.isUsageExcludable`.

Reply:
Well spotted! I updated it to explain what's actually happening now :)
Thank you!